src/main/java/cs2103/v15_1j/jimjim/model/Task.java

```diff
@@ -1,47 +1,74 @@
 package cs2103.v15_1j.jimjim.model;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
 
 import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.ObjectProperty;
 import javafx.beans.property.SimpleBooleanProperty;
-import javafx.beans.property.SimpleStringProperty;
-import javafx.beans.property.StringProperty;
+import javafx.beans.property.SimpleObjectProperty;
 
-public class Task {
+public class Task extends TaskEvent implements Comparable<Task> {
 
-	private final StringProperty taskName;
-	private final BooleanProperty completed;
+	private ObjectProperty<LocalDateTime> dateTime;
+	private ObjectProperty<LocalDate> date;
```

Comment:
I need to use a DatePicker and it can only bind directly to a LocalDate object. I tried to just use LocalDateTime but the automatic binding wouldn't work.